src/components/NewCertificate.tsx

```diff
@@ -0,0 +1,181 @@
+      alert('Certificate not added');
+    }
+  };
+  const resetFields = () => {
+    setSupplier('');
+    setCertificateType('');
+    setValidTo('');
+    setValidFrom('');
+    setPreview(undefined);
+  };
+
+  //function to handle the File (pdf) upload
+  const handleFile = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0];
+    if (file && file.type === 'application/pdf') {
+      const reader = new FileReader();
+      reader.onloadend = () => {
+        setPreview(reader.result as string);
+      };
+      reader.readAsDataURL(file);
+    } else {
+      alert('Only Pdf are to be select, thank you!');
+    }
+  };
+
+  return (
+    <div className="new-cert-form">
+      <form onSubmit={handleSaving}>
+        <div className="form-container">
+          <div className="left-side">
+            <div className="form-input-container">
+              <label className="form-input-label">Supplier</label>
+              <div className="form-input-container form-input-multiple">
+                <input
+                  type="text"
+                  className="form-input"
+                  value={supplier}
+                  onChange={(e) => setSupplier(e.target.value)}
+                />
+                <div>
+                  <IconSvg
+                    Icon={searchIcon}
+                    className="input-icon input-icon-search"
+                  />
+                  <div className="vertical-bar"></div>
+                  <IconSvg
+                    Icon={closeIcon}
+                    className="input-icon"
+                  />
+                </div>
+              </div>
+            </div>
+            <div className="form-input-container">
+              <div className="form-input-container">
+                <label className="form-input-label">Certificate type</label>
+                <select
+                  name="suppliers"
+                  id="suppliers"
+                  className="form-input form-input-select"
+                  value={certificateType}
+                  onChange={(e) => setCertificateType(e.target.value)}
+                >
+                  <option value="">Select your option</option>
+                  <option value="Printing of Permission">
+                    Printing of Permission
+                  </option>
+                  <option value="Printing of Permission">
+                    Printing of Permission
+                  </option>
+                  <option value="Printing of Permission">
+                    Printing of Permission
+                  </option>
+                </select>
+              </div>
+            </div>
+            <div className="form-input-container">
+              <label className="form-input-label">Valid from</label>
+              <input
+                type="date"
+                className="form-input"
+                value={validFrom}
+                onChange={(e) => setValidFrom(e.target.value)}
```

Comment:
Please fix all of inline functions, this was already explained in previous review.